src/plugins/home/public/application/components/home.js

```diff
@@ -124,58 +124,55 @@ export class Home extends Component {
     }
 
     return (
-      <main
-        aria-labelledby="kbnOverviewPageHeader__title"
-        className="homWrapper"
-        data-test-subj="homeApp"
+      <EuiPageTemplate
```

Comment:
Can you actually use the Kibana version here? Trying to use that one every where so that we can update one place with kibana-specific features.

Also thoughts on using the `template="empty"` version so that there's no white background behind all the cards?

Reply:
https://github.com/elastic/kibana/blob/master/dev_docs/tutorials/kibana_page_template.mdx

Reply:
Sure thing. Updated here: https://github.com/elastic/kibana/pull/103003/commits/1d665ba17d1f087ff1c561989692fee08503373b

Reply:
Awesome, yeah that's better! Thanks!